README.md

```diff
@@ -316,7 +322,7 @@ Alchemy specs are written **in RSpec 3**. Please **do not use deprecated RSpec 2
 * Read the guidelines: http://guides.alchemy-cms.com.
 * Read the documentation: http://rubydoc.info/github/AlchemyCMS/alchemy_cms
 * If you found a bug please use the [issue tracker on Github](https://github.com/AlchemyCMS/alchemy_cms/issues).
-* For questions about general usage please use [Stack Overflow](http://stackoverflow.com/questions/tagged/alchemy-cms), [the User Group](http://groups.google.com/group/alchemy-cms) or the [IRC channel](irc://irc.freenode.net#alchemy_cms).
+* For questions about general usage please use [Slack](https://alchemy-cms.slack.com), [Stack Overflow](http://stackoverflow.com/questions/tagged/alchemy-cms), [the User Group](http://groups.google.com/group/alchemy-cms) or the [IRC channel](irc://irc.freenode.net#alchemy_cms).
```

Comment:
Please do not replace the "Stack Overflow" notice. 

You could instead replace the IRC one.

Reply:
What do you mean?
I did not replace the "Stack Overflow", it exists after the "Slack"...
Can you please explain more?
Thank you

Reply:
Sorry. You are right. I misread this. Please replace the IRC part with the Slack link.

Reply:
Done...
I would suggest changing also the label...

Reply:
Yes. Sure

Reply:
Done.